content/en/community/_index.html

```diff
@@ -121,7 +121,7 @@ <h1 style="padding-top:1%">Discussions</h1>
 <div class="resourcecontainer">
 
 <div class="resourcebox">
-<img src="/images/community/discuss.png" style="width:80%;padding-bottom:2%">
+<img src="/images/community/discuss.png" alt="discuss" style="width:80%;padding-bottom:2%">

```

Comment:
Nit: <img src="/images/community/discuss.png" alt="Forum" style="width:80%;padding-bottom:2%">

We are yet to draft the guidelines for alt text. However, please refer to https://webaim.org/techniques/alttext/#functional

Reply:
Included this in latest commit